stdlib/encodings/utf_8_sig.pyi

```diff
@@ -0,0 +1,27 @@
+import codecs
+
+class IncrementalEncoder(codecs.IncrementalEncoder):
+    def __init__(self, errors: str = ...) -> None: ...
+    def encode(self, input: str, final: bool = ...) -> bytes: ...
+    def reset(self) -> None: ...
+    def getstate(self) -> int | str: ...
+    def setstate(self, state: int | str) -> None: ...
+
+class IncrementalDecoder(codecs.BufferedIncrementalDecoder):
+    def __init__(self, errors: str = ...) -> None: ...
+    def _buffer_decode(self, input: bytes, errors: str | None, final: bool) -> tuple[str, int]: ...
```

Comment:
Is there a reason to add this? We usually don't include non-public methods in stubs.

Reply:
because it was included in `utf_8.py` so figured it made sense to be consistent? Happy to remove if you want though?

Reply:
We can keep it for consistency.